packages/eslint-plugin/docs/rules/naming-convention.md

```diff
@@ -41,7 +40,35 @@ type Options = {
   types?: Types<Selector>[];
 }[];
 
-const defaultOptions: Options = [];
+// the default config essentially does the same thing as ESLint's camelcase rule
+const defaultOptions: Options = [
+  {
+    selector: 'default',
+    format: ['camelCase'],
+  },
+
+  {
+    selector: 'variable',
+    format: ['camelCase', 'UPPER_CASE'],
+  },
+  {
+    selector: 'parameter',
+    format: ['camelCase'],
+    leadingUnderscore: 'allow',
+  },
+
+  {
+    selector: 'memberLike',
+    modifiers: ['private'],
+    format: ['camelCase'],
+    leadingUnderscore: 'require',
+  },
```

Comment:
Is this supposed to match `defaultCamelCaseAllTheThingsConfig`? It doesn't include handling of private members.

Reply:
oops. I deleted that from the default config and forgot to sync it here. thanks